release/release.go

```diff
@@ -1576,7 +1576,16 @@ func downloadArtifacts(v string, artifactNames []string) {
 	check(err, "os.Getwd")
 	fmt.Printf("pwd: %s\n", pwd)
 
-	_, err = run("git", "clone", "git@github.com:10gen/mongo-release.git")
+	token := os.Getenv("generated_token_mongo_release")
+	if token == "" {
+		log.Fatalf("invalid empty token")
+	}
```

Comment:
```suggestion
        const tokenVar := "generated_token_mongo_release"
	token := os.Getenv(tokenVar)
	if token == "" {
		log.Fatalf("The environment must contain a nonempty %#q.", tokenVar)
	}
```

Reply:
Nit: It’d be nice to make this a separate function here to facilitate easier adaptation to mongosync.